### DISCUSSION TITLE
Unfair Automatic Rejection - GitHub Education

### DISCUSSION THREAD

**Discussion by MaliciousFiles** [community/community#54069] [GitHub Education]
I am a sophomore at the Liberal Arts and Science Academy (LASA) High School in Austin, TX. Notably, my school is autocompleted in their form so it is recognized. I first applied for GitHub Education on Oct 27, 2021 in my freshman year, by submitting my school ID and got accepted:

![image](https://user-images.githubusercontent.com/57738870/235333707-9da5e0ad-ded5-4e3f-b4b8-6d31eb6baa9a.png)

It wasn't due to expire until Oct 27, 2023. However, I recently started working on a group project for my Advanced CS class and wanted my fellow groupmates to be able to use the CLion Jetbrains IDE. I have been using the Jetbrains suite ever since I got GitHub Education and have loved it. However, when they applied, also using their school IDs, they kept getting auto-denied with the following suggestion:

"Try applying again after making more [contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution) to the GitHub Classroom assignment(s) created by your teacher."

When I first applied I wasn't part of any GitHub Classroom, so this confused me quite a bit. This year, however, we have one for class, but hardly ever use it. We've had 3 total assignments, with maybe 1-5 commits each. My initial thought was to Google for anyone in the same boat and came across [this discussion](https://github.com/orgs/community/discussions/52752). They evidently didn't find an answer.

To try and play around with it myself, I clicked the "Reverify your academic affiliation" button on my portal. Filling out my information, with my school ID, I then got denied for the same reason. Then the most idiotic part of this debacle revealed itself: at the same time as receiving the rejection email, I received the following:

![image](https://user-images.githubusercontent.com/57738870/235334072-9f59f245-d803-429a-a97a-24b9568be2af.png)

The billing page also now shows that my "coupon" is expiring today:

![image](https://user-images.githubusercontent.com/57738870/235334739-4fd0ea7a-1132-4642-b6b4-f8ac7fa5e670.png)

Now I was annoyed. I am still a student at the same school as when I was accepted previously, and I submitted everything the exact same way. I looked for any way to contact support, and apparently not only is there no email, but *they aren't even accepting support tickets for getting rejected anymore*. So, I decided screw it; I'm going to try to game the system. I set up a quick PowerShell  script to farm commits on a GitHub Education classroom, because apparently they're trying to tell me that I didn't student enough.

That appears to have worked because the suggestion to contribute more went away. However, my problem isn't solved, it just gave up on giving me suggestions:

![image](https://user-images.githubusercontent.com/57738870/235334302-5dcbe88d-3ac7-46be-b4f0-635833660478.png)

I scoured these forums, looked through every FAQ link I could find, and tried to find any way I could talk to an actual GitHub employee (an email, a support ticket, a discord, anything). What can I do? What's my recourse in this new, automated system? Any help would be appreciated before I lose access to the IDEs I've been using for over a year.

**Comment by samus-aran** [comment]
Hi @MaliciousFiles 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
